app/views/layouts/_footer.html.erb

```diff
@@ -1,15 +1,29 @@
-<footer>
-  <div class="home-footer footer navbar-bottom">
-    <div class="home-footer-left">
-      <div><%= image_tag "logo.png" %></div>
-      <div><%= link_to "About", about_path %></div>
-      <div><%= link_to "Contact", contact_path %></div>
-      <div><%= link_to "FAQ", faq_path %></div>
+<footer class="footer">
+    <div class="container">
+        <div class="row">
+            <div class="footer-logo col-sm-2 col-xs-12">
+                <a class="navbar-brand" href="/"><img class="logo-nav logo-footer" src="/images/logo0.png" alt="Arcade"/></a>
+            </div>
+            <div class="footer-list footer-list1 col-sm-2 col-xs-4 text-center">
+                <%= link_to "About", about_path %>
+            </div>
+            <div class="footer-list col-sm-2 col-xs-4 text-center">
+                <%= link_to "Contact", contact_path %>
+            </div>
+            <div class="footer-list col-sm-2 col-xs-4 text-center">
+                <%= link_to "FAQ", faq_path %>
+            </div>
+            <div class='footer-social col-xs-12 col-sm-4'>
+                <center>
```

Comment:
The `center` tag is deprecated (https://developer.mozilla.org/en-US/docs/Web/HTML/Element/center). Instead wrap it in a `section` tag. Maybe, `<section class="footer-social-icons"> ...` and use `display: flex` on it to center the icons.